packages/keystone/plugins/utils/typing.js

```diff
@@ -14,6 +14,7 @@ const TEST_EXAMPLE = {
 }
 
 function assertPluginCall (fn) {
+    // NOTE: we want to check that the plugin is a pure function! It should not save or use any global state!
     fn(TEST_EXAMPLE, { schemaName: 'Test', addSchema: () => undefined })
```

Comment:
Could you please explain in comments what is a TEST_EXAMPLE
I quess, this is a stub of an input data for a plugin, accepting that the function should not fail.